Gruntfile.js

```diff
@@ -45,6 +45,12 @@ module.exports = function (grunt) {
                 ],
                 dest: directoryPublic + '/edit.less',
             },
+            cssTextEditor: {
+                src: [
+                    directoryPrivate + '/text_editor.less',
```

Comment:
The idea here was that each view should only have one CSS/JS file, by concatenating multiple files together. 

We can plug that in here, which will simplify things and be slightly faster.

Just add this `directory + '/text/editor.less` bit as an item in the `src` arrays of both `cssView` and `cssEdit`.

```javascript
{
    cssView: [
        directoryPrivate + '/view.less',
        directoryPrivate + '/text_editor.less',
    ],
}
```

Then you can delete the `cssTextEditor` block here.